CHANGELOG.md

```diff
@@ -1,3 +1,82 @@
+# Announcing the release of OSMT 3.0.0
+June 20, 2023
+## Enhancements
+OSMT 3.0.0 brings 6 new feature enhancements
+
+- Versioned API routes to support backward compatibility with legacy integrations (pre-OSMT 3.0)
+
+- Skills (RSDs) now support multiple categories
+
+- Skills (RSDs) now support multiple authors
+
+- Search result filtering
+
+- Selectable pagination quantity
+
+- Export to Excel Workbook .xlsx option (previously only CSV)
+
+## Updates
+
+- Updated UI to Angular v16 (previously v12)
+
+- Updated other dependencies
+
+## Developer/Integrator Enhancements
+
+- OpenAPI compliant API specification
+
+- New API test suite
+
+## Model Changes:
+- RSD 'category' -> 'categories'
+
+- RSD 'author' -> 'authors'
+
+## Defect Fixes
+- Removed leading `'-'` character from URLs in export files
+
+## Removed
+- Protractor e2e (end-to-end) testing framework (deprecated & unused by OSMT)
```

Comment:
TSLint & codelyzer were also removed, we can add that only if we want to be more detailed.